examples/viewcopy/viewcopy.cpp

```diff
@@ -472,11 +191,19 @@ set ylabel "throughput [GiB/s]"
 #endif
     benchmarkMemcpy(
         "memcpy(p)",
-        [&](auto* dst, auto* src, auto size) { parallel_memcpy(dst, src, size, std::memcpy, numThreads); });
+        [&](auto* dst, auto* src, auto size)
+        {
+#pragma omp parallel
```

Comment:
This will hopefully get solved with clang-format 12.

Reply:
LLAMA already uses clang-format 12 for several weeks :D But what do you think needs to be solved here?

Reply:
The `#pragma` should be indented.

Reply:
Ah, I see! Well that is for sure something clang-format can handle. But I am using the CRP `.clang-format`, so I guess the coding guidelines don't indent it.